### ISSUE TITLE
Errors back from qonsole

### ISSUE THREAD

**Issue by ijdickinson** [epimorphics/hmlr-linked-data#2]
Via Dave Lockett, but reported by customer:

> Dear Sirs, 
> Hope you are well. 
> We have emailed a few times regarding this issue but have not received any response 
> as yet. Could you kindly provide an update whether the issue is being looked into and 
> when it is likely to be resolved if possible?

> The console (or API in general) seems to be down. Some of the calls made through the 
> console (https://landregistry.data.gov.uk/app/qonsole) are successful but most of the time, 
> are return the following error message:

```html
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
...
```

**Comment by ijdickinson** [comment]
@der will investigate initially. Thanks Dave!

**Comment by der** [comment]
Sorted. This was a configuration error left from when I built the new server last week. 

Turns out the chef script for the qonsole config will only work first time, but when the server first comes up we don't know what data tier to connect to so it defaulted to localhost. The process I used of updating the chef node configuration and then running chef again to tie everything up worked for everything except the qonsole config which stayed on localhost.

No idea why the chef recipe is written that way and why the apache config doesn't allow localhost in the first place.
